integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSource.scala

```diff
@@ -104,14 +106,17 @@ class CarbonSource extends CreatableRelationProvider
   }
 
   private def createTableIfNotExists(sparkSession: SparkSession, parameters: Map[String, String],
-                                     dataSchema: StructType): String = {
+      dataSchema: StructType): String = {
 
     val dbName: String = parameters.getOrElse("dbName", CarbonCommonConstants.DATABASE_DEFAULT_NAME)
-    val tableName: String = parameters.getOrElse("tableName", "default_table")
+    val emptyTableName: String = parameters.getOrElse("tableName", "default_table")
+    if (StringUtils.isBlank(emptyTableName)) {
+      throw new MalformedCarbonCommandException("INVALID TABLE NAME")
```

Comment:
I think there is misunderstanding, I suggest to give "the specified table name is blank" in the exception message. Not suggesting to change the variable name.